site/docs/v1/tech/apis/identity-providers/_samlv2-response-body.adoc

```diff
@@ -86,6 +86,22 @@ include::_identity-provider-linking-strategy-response-parameter.adoc[]
 [field]#identityProvider.name# [type]#[String]#::
 The name of this SAML v2 identity provider. This is only used for display purposes.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.nameIdFormat# [type]#[String]#::
```

Comment:
Is this part of IdP initiated?

Reply:
This is for an `ifndef` which means that we will exclude this field for SAML Idp init.  I believe that is correct.